CHANGELOG.md

```diff
@@ -17,18 +17,18 @@
 
 - The FreeRTOS-IoT-Libraries directories in the previous snapshot version (V2020218_LTS_development_snapshot) under FreeRTOS-Plus/Source and FreeRTOS-Plus/Demo has been renamed to *FreeRTOS-IoT-Libraries-Beta1* to make way for a parallel directory, *FreeRTOS-IoT-Libraries-Beta2*, that holds libraries for which the refactoring work is near completion (currently the MQTT library).
 
-#### New MQTT Client Library (Beta 2)
+#### MQTT LTS RC1 Client Library (in FreeRTOS-IoT-LTS-Beta2)
 
 - A new MQTT library source code has been added under the FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-Beta2 directory.
```

Comment:
Please change to some along the lines of "The first release candidate for the refactored MQTT library is located in the FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-Beta2 directory.  The library will be released in the main FreeRTOS download when the refactoring described on the LTS roadmap page is complete".

Reply:
"refactoring describe on the LTS roadmap page" can be ambiguous, especially, as there is no mention of "refactoring" on the lts-roadmap page

Reply:
How about: "The first release candidate for the refactored MQTT library is located in the FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-Beta2 directory. The library will be released in the main FreeRTOS download when the work described on the LTS roadmap page on the FreeRTOS.org is complete".

Reply:
Sorry - didn't realise this had been done already - just approved.